.shellcheckrc

```diff
@@ -0,0 +1,9 @@
+# Shellcheck configuration file
+
+# SC1090: Can't follow non-constant source. Use a directive to specify location
+# Ignore until https://github.com/koalaman/shellcheck/issues/769 is fixed upstream
+disable=SC1090
+
+# SC2039: In POSIX sh, local is undefined
+# OpenRC allows local, which isn't POSIX
```

Comment:
link to https://github.com/koalaman/shellcheck/issues/1341

Reply:
Added a link, please let me know if you'd like to tweak the comment.